modules/atomic-widgets/styles/styles-renderer.php

```diff
@@ -126,6 +134,10 @@ private function props_to_css_string( array $props ): string {
 		return Collection::make( Props_Resolver::for_styles()->resolve( $schema, $props ) )
 			->filter()
 			->map( function ( $value, $prop ) {
+				if ( $this->after_prop_transform ) {
+					call_user_func( $this->after_prop_transform, $prop, $value );
```

Comment:
```suggestion
					call_user_func( $this->after_prop_transform, $prop, $value, $schema[ $prop ] );
```

maybe?
so you can check the type when using it, as I said above

Reply:
I think checking based on the key is better